lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -1,8 +1,17 @@
 <template>
-	<div class="badge" :class="{ '-neutral': color === BADGE_COLORS.NEUTRAL }">
-		<div class="badge__slot">
-			<slot />
-		</div>
+	<div
+		class="badge"
+		:class="{
+			'-neutral': color === BADGE_COLORS.NEUTRAL,
+			'-primary': color === BADGE_COLORS.PRIMARY,
+		}"
+	>
+		<ds-icon v-if="icon" class="badge__icon" :icon="icon" :size="ICON_SIZES.XX_SMALL" />
+		<template v-if="$slots.default">
```

Comment:
why don't you put `v-if on `<div>`?

Reply:
I thought it has to be templated in both cases

Reply:
nope, you can use it on `<div>`, the same way as on `<ds-icon>` line above